create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -36,50 +44,52 @@ include::modules/op-resolver-hub-config.adoc[leveloffset=+2]
 include::modules/op-resolver-hub.adoc[leveloffset=+2]
 
 [id="resolver-bundles_{context}"]
-== Specifying a remote pipeline or task from a Tekton bundle
+== Specifying a remote pipeline, task, or step action from a Tekton bundle
```

Comment:
```suggestion
== Specifying a remote pipeline, task, or StepAction from a Tekton bundle
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".